tests/test_evaluation.py

```diff
@@ -5,10 +5,14 @@
 
 from graphrag_eval import (
     compute_aggregates,
-    evaluate_steps,
     run_evaluation,
     stats_for_series,
 )
+from graphrag_eval.steps.evaluation import evaluate_steps
```

Comment:
I find it strange and incosistent that we don't import this with `from graphrag_eval import evaluate_steps` . In the __init__ we have
```
from .aggregation import *
from .evaluation import *
from .steps import *
from .steps.sparql import *
```

Do you think we should expose only the two functions `compute_aggregates` and `run_evaluation` or everything?

Reply:
This comment is actually about file `graphrag_eval/__init__.py`, right?

Yes, let's limit what we expose. I now modified  `graphrag_eval/__init__.py` to import only `run_evaluation` and `compute_aggregates`. If we want to expose other functions, we should rename them, because `get_*_evaluation_dict` are clumsy.